lib/wsdl.js~

```diff
@@ -0,0 +1,2198 @@
+/*
```

Comment:
please remove this file (it seems like this is a backup file from your Editor/IDE?)